boilerplate/.gitignore.ejs

```diff
@@ -40,13 +39,28 @@ yarn-error.log
 buck-out/
 \.buckd/
 *.keystore
+!debug.keystore
 
-# VS Code
-.vscode
-
-# Cocoapods
-/ios/Pods
+# fastlane
+#
+# It is recommended to not store the screenshots in the git repo. Instead, use fastlane to re-generate the
+# screenshots whenever they are needed.
+# For more information about the recommended setup visit:
+# https://docs.fastlane.tools/best-practices/source-control/
 
+*/fastlane/report.xml
+*/fastlane/Preview.html
+*/fastlane/screenshots
 
 # Bundle artifact
 *.jsbundle
+
+# CocoaPods
+/ios/Pods/
+
+# Bowser-specific items
+# You can safely replace everything above this comment with whatever is
+# in the default .gitignore generated by React-Native CLI
+
+# VS Code
+.vscode
```

Comment:
This is our only customization of this file. Perhaps we should let RN generate it and then just append.